pywbem/tupleparse.py

```diff
@@ -1615,7 +1639,8 @@ def parse_ireturnvalue(tup_tree):
                                 VALUE.OBJECTWITHLOCALPATH* | VALUE.OBJECT* |
                                 OBJECTPATH* | QUALIFIER.DECLARATION* |
                                 VALUE.ARRAY? | VALUE.REFERENCE? | CLASS* |
-                                INSTANCE* | VALUE.NAMEDINSTANCE*)>
+                                INSTANCE* | VALUE.NAMEDINSTANCE* |
+                                VALUE.INSTANCEWITHPATH)>
```

Comment:
I was wondering why not zero to many of VALUE.INSTANCEWITHPATH, and it turns out that DSP0201 allows only one, while the Open and Pull ops defined in DSP0200 need zero to many. Please double check, but I believe we have an issue for DSP0201 here.